src/Sylius/Behat/Service/SharedStorage.php

```diff
@@ -22,7 +24,7 @@ class SharedStorage implements SharedStorageInterface
     public function get(string $key)
     {
         if (!isset($this->clipboard[$key])) {
-            throw new \InvalidArgumentException(sprintf('There is no current resource for "%s"!', $key));
+            throw new SharedStorageElementNotFoundException($key);
```

Comment:
👌

Similar should be made for `::getLatestResource` or just 2 factory methods within the exception 🤷